### PR TITLE
l10n Occitan: theme site, custom emoji, Oauth, newcomers follow admins, ...

### PR DESCRIPTION
**Pull request by JeanGauthier**

### CODE REVIEW

**Comment by JeanGauthier** [comment]
I don't know what to change so it works :S 
Is it on line 186?

File: config/locales/oc.yml [hunk 1]
```diff
@@ -517,10 +539,13 @@ oc:
     settings: Paramètres
     two_factor_authentication: Autentificacion en dos temps
     your_apps: Vòstras aplicacions
+themes:
```
Comment (ykzts) [inline config/locales/oc.yml]: add indent

**Comment by ykzts** [comment]
cc @Quent-in

File: config/locales/oc.yml [hunk 2]
```diff
@@ -190,6 +209,9 @@ oc:
         desc_html: Afichada sus la pagina de las condicions d’utilizacion<br>Podètz utilizar de balisas HTML
         title: Politica de confidencialitat del site
       site_title: Títol del site
+      thumbnail:
+         desc_html: Servís pels apercebuts via OpenGraph e las API. Talha de 1200x630px recomandada.
```
Comment (ykzts) [inline config/locales/oc.yml]: remove one whitespace.

**Review by ykzts — DISMISSED** [review]

File: config/locales/oc.yml (hunk 1 above)
Comment (JeanGauthier) [reply config/locales/oc.yml]: Done, it worked! Thank you for pointing where I had to make changes!

**Review by Gargron — APPROVED** [review]

**Pull request merged**